.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: rust
+rust:
+  - stable
+  - beta
```

Comment:
I'm not sure I fully understand the advantages of building/running tests against `beta`, especially since `nightly` feeds into `beta` and `beta` feeds into `stable`.

Reply:
Travis's docs mentioned that "_The Rust team appreciates testing against the beta and nightly channels, even if you are only targeting stable._"

Digging a little bit deeper, [outdated docs](https://doc.rust-lang.org/book/first-edition/release-channels.html#helping-the-ecosystem-through-ci) mention that testing `beta` and `nightly` helps to "_catch regressions even sooner_".

[Current docs (2018)](https://doc.rust-lang.org/book/2018-edition/appendix-06-nightly-rust.html#appendix-f---how-rust-is-made-and-nightly-rust) do not mention anything about building against the three release channels - weird.

Reply:
Interesting bit from the (outdated) First Edition:

> Additionally, testing against nightly can catch regressions even sooner, and so if you don’t mind **a third build**, we’d appreciate testing against all channels.

Emphasis mine, it seems they prioritize `beta` over `nightly`.

I suppose I'm more used to working with `stable`/`nightly` because some of the libraries I use for other projects require `nightly` features to compile.

I'm good to test against all 3, we'll let Travis handle the workload 😁